src/main/java/org/jitsi/videobridge/octo/OctoTentacle.java

```diff
@@ -353,6 +409,13 @@ public JSONObject getDebugState()
         debugState.put("relay", relay.getDebugState());
         debugState.put("targets", targets.toString());
 
+        /* TODO: do we really want to dump debug state for all the queues? */
```

Comment:
Good question.  Should we just sum up the stats?

Reply:
If I can get my queue stats patch in a usable state, we can get global state for the octo send queue.

Absent that, I don't think there's a whole lot of value for the non-statistics-enabled PacketQueue debug state per queue.